### DISCUSSION TITLE
How did the team come to a conclusion on the Paddle OCR chinese characters - ppocr_keys_v1.txt ?

### DISCUSSION THREAD

**Discussion by saikrishna431** [PaddlePaddle/PaddleOCR#12963] [Q&A]
Hi 
I would like to understand what is the base for selecting the characters for Chinese language Vocab file ppocr_keys_v1.txt. 

I see that the characters are combination of chinese Simplified and Traditional.

**Comment by tink2123** [comment]
This dictionary selects common characters in Chinese to ensure that most scenarios are covered.

**Comment by saikrishna431** [comment]
The total Number of characters in Chinese are more than 20000. Out of which only 6600 characters are selected.  Can you elaborate on how did we select these characters. 
And is there any master list of characters from which the Vocab is selected in ppocr_keys_v1.txt
